### DISCUSSION TITLE
Table display is broken

### DISCUSSION THREAD

**Discussion by sazalahsan** [rappasoft/laravel-livewire-tables#1718] [Q&A]
Hi, the table display is not showing as expected. Initially it does not show search box and column option. If I go to next page, they are shown but column gets broken. If I return to first page again, half broken. Very strange behavior. What am I missing? Any help is appreciated

<img width="1066" alt="Screenshot 2024-05-17 at 1 17 38 AM" src="https://github.com/rappasoft/laravel-livewire-tables/assets/20285210/efaf0305-5c15-45ec-acac-a37e96acd93d">
<img width="1066" alt="Screenshot 2024-05-17 at 1 17 56 AM" src="https://github.com/rappasoft/laravel-livewire-tables/assets/20285210/d7ff352e-3ddd-4f7d-a53c-ef9ce4ac7e90">
<img width="1066" alt="Screenshot 2024-05-17 at 1 18 07 AM" src="https://github.com/rappasoft/laravel-livewire-tables/assets/20285210/34dfeb02-625d-45c7-8930-2e84f8417185">

**Comment by lrljoe** [comment]
Looks like the javascript isn't loaded in correctly, try the bundle approach (see the docs)

Check your browser console for any errors that are popping up
